react/features/device-selection/components/DeviceSelector.js

```diff
@@ -155,16 +157,22 @@ class DeviceSelector extends Component {
         const triggerText
             = (options.defaultSelected && options.defaultSelected.content)
                 || options.placeholder;
+        const trigger = this._createDropdownTrigger(triggerText);
+
+        if (options.isDisabled) {
```

Comment:
The same as above. If this needs to be fixed in AtlasKit, let's report to them and add a FIXME comment here explaining why we do it.